.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+## PR Checklist
+Please check that your PR fulfills the following requirements:
+
+- [ ] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+
+
+## PR Type
+What kind of change does this PR introduce? 
+
+<!-- Please check the one that applies to this PR using "x". -->
+```
+[ ] Bugfix
```

Comment:
don't leave space

Reply:
Could you be specific :slightly_smiling_face: